tests/test_mock_command_handler.py

```diff
@@ -130,15 +129,16 @@ async def test_configure(self) -> None:
         await self.command_handler.stop_sending_telemetry()
 
     async def test_start_and_stop_sending_telemetry(self) -> None:
-        with pytest.raises(common.CommandError) as cm:
-            await self.command_handler.stop_sending_telemetry()
-        command_error = cm.value
-        assert command_error.response_code == common.ResponseCode.NOT_STARTED
+        # This command should be silently ignored.
+        assert len(self.command_handler.devices) == 0
+        await self.command_handler.stop_sending_telemetry()
```

Comment:
Would it make any sense to have stop_sending_telemetry return a flag to indicate if there was telemetry to stop? That must depend on how your command handlers work in general and it may well not be worth the bother. But "silently ignore" makes me a bit uneasy.

Actually, I think it logs something, in which case you could use self.assertLogs here (or the pytest equivalent, but I've found it much clumsier than the unittest version).

Reply:
I opted for `self.assertLogs`.